packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -278,10 +297,13 @@ function SendConfirmation(props: Props) {
         dekFee = fee.dividedBy(2)
       }
 
-      ValoraAnalytics.track(FeeEvents.fee_rendered, {
-        feeType: 'Security',
-        fee: securityFee ? securityFee.toString() : securityFee,
-      })
+      // DO NOT MERGE: Test this out.
```

Comment:
reminder

Reply:
Thanks! I went ahead and tested this manually and it seems to work as intended. (Not sure how I might add an automated test)